SR2021/2021-01-01-january-update.md

```diff
@@ -0,0 +1,59 @@
+---
+to: SR2021 teams
+subject: Student Robotics January Update
+---
+
+Hello there!
+
+There are less than 2 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and give you an update.
+
+## League Sessions
+
+The first league session will be happening on the 16th of January, at 12:00 UK time, lasting around 1.5 hours. You will need to have submitted your code to us (more on that below) by 08:00 UK time on the same day. If you do not submit code, or submit late, you will be marked absent and score 0 points.
+
+Further details, and a link to the livestream, will be on our website soon - for this and the other league sessions.
+
+## Simulator Update
+
+We released a new version of the simulator, which adds a few important improvements:
+
+- Support for Webots 2021a (no more deprecation warnings).
+- Add several scientific libraries. See [the docs](https://studentrobotics.org/docs/programming/python/libraries) for a full list.
+- Reconstructed arena to fix issues with distance sensors and shadows. **Note**: the new arena is _slightly different_ now, mostly that the interior (diagonal) walls have moved by a few centimetres.
+- The error message about rate-limited LEDs now includes the ruggeduino pin number for the LED, rather than its name.
+
+This can be downloaded from [the docs](https://studentrobotics.org/docs/simulator/#installing-the-simulation). We recommend updating and testing your code with this version as soon as possible. Both the first league session and friendlies will use this version.
+
+## Friendlies
+
+As noted in our previous email, we're going to be running some friendly matches on the 9th of January. These matches are entirely optional, and will not contribute to the league. The matches are simply a chance to see how your robot performs against other robots, as well as socialise with other teams and us volunteers. Unlike our other events, these matches will be incredibly relaxed and informal. We encourage anyone to join in the conversation in Discord or in the chat.
+
+[Watch our livestream](https://youtu.be/iwa-7R6sR8M).
```

Comment:
Suggest inlining the livestream link? I _think_ this makes it feel a bit less formal, which seems to be the goal of the friendlies.

```suggestion
As noted in our previous email, we're going to be running some friendly matches on the 9th of January. These matches are entirely optional, and will not contribute to the league. The matches are simply a chance to see how your robot performs against other robots, as well as socialise with other teams and us volunteers.

We'll use a YouTube [livestream](https://youtu.be/iwa-7R6sR8M) to broadcast the matches and as usual we'll provide your logs afterwards. Unlike our other events, these matches will be incredibly relaxed and informal. We encourage anyone to join in the conversation in Discord or in the chat.
```